ui/job-view/GeckoProfile.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```

Comment:
I think this file is unused? (I presume leftover from before the in-tree task was created?)

Reply:
thanks, I had originally had that file and forgot to 'git rm'; apologies for sloppy pushes

Reply:
Not a problem at all! Just though it would give Cameron less to review :-)